src/translations/screens/subscreens/TravelToken.ts

```diff
@@ -700,9 +710,9 @@ export default orgSpecificTranslations(SelectTravelTokenTexts, {
     },
     toggleToken: {
       title: _(
-        'Bytt mellom reisekort / mobil',
-        'Switch between travel card / phone',
-        'Byt mellom reisekort / mobil',
+        'Bytt mellom reisekort/telefon',
+        'Switch between travel card/phone',
+        'Byt mellom reisekort/telefon',
```

Comment:
In other places, we have a space between the /

```suggestion
        'Bytt mellom reisekort / telefon',
        'Switch between travel card / phone',
        'Byt mellom reisekort / telefon',
```

Reply:
Yeh.. This is an overwhelming issue/bug in the entire translation tree. Språkrådet says there should be no [space between single words in listing of single word alternatives](https://sprakradet.no/godt-og-korrekt-sprak/rettskriving-og-grammatikk/tegn/skrastrek/). Though, to have this typo being consistent, I'll 'fix' this